lib/cfg-grammar.y

```diff
@@ -332,13 +332,14 @@ extern struct _StatsOptions *last_stats_options;
 %token KW_KEY                         10504
 %token KW_SCOPE                       10505
 %token KW_SHIFT                       10506
-%token KW_REKEY                       10507
-%token KW_ADD_PREFIX                  10508
-%token KW_REPLACE_PREFIX              10509
+%token KW_SHIFT_LEVELS                10507
```

Comment:
Just a minor thing but I think changing these id's will break the binary ABI. If a maintainer wants to port it back, all dependant modules need to be recompiled.

Reply:
true, but I don't think we have ABI guarantees.